flocker/node/_change.py

```diff
@@ -62,16 +62,33 @@ def run_state_change(change, deployer):
 
     :return: ``Deferred`` firing when the change is done.
     """
+    if isinstance(change, _InParallel):
+        return gather_deferreds(list(
```

Comment:
Since it works and it's just a refactoring, I'll probably leave it as-is.  I can imagine revisiting this decision at some future point though.